bcgov/HOUS-permit-portal
# Merge fix-error-on-file-map and add jurisdiciton select with PIN mode…

Sorry for the git ammend over Sunnys merge commit

## Description

support PIN and jurisdicitonless (federal) land addresses (first nations)

## What type of PR is this? (check all applicable)

- [ x] 🍕 Feature

## Related Tickets & Documents

https://hous-bssb.atlassian.net/browse/BPHH-972

## Steps to QA

log in as submitter
try to create a permit application with the address:

3090 Takaya Dr

The lack of a jurisdiction should be detected and the user should get the opportunity to enter a PIN and search for a jurisdiction
